src/manifest.ts

```diff
@@ -58,13 +59,27 @@ export class SlackManifest {
       }, {} as ManifestSchema["types"]);
     }
 
+    //TODO: Add support for datastores
```

Comment:
I think this is a relic from merging/ rebasing `main`. Datastore support is directly below :)
```suggestion
```

Reply:
🦅 👁️